pkgs/applications/terminal-emulators/wezterm/default.nix

```diff
@@ -42,6 +42,10 @@ rustPlatform.buildRustPackage rec {
     hash = "sha256-Az+HlnK/lRJpUSGm5UKyma1l2PaBKNCGFiaYnLECMX8=";
   };
 
+  cargoPatches = [
+    ./bump-time-crate.diff
+  ];
+
```

Comment:
Maybe it would be better to link the Cargo.lock into place like other packages do, so we don't need to commit a Cargo.lock that ends up being patched?